config/default.yml

```diff
@@ -1461,6 +1461,9 @@ Lint/UnusedMethodArgument:
   AllowUnusedKeywordArguments: false
   IgnoreEmptyMethods: true
 
+Lint/Void:
+  NonmutatingMethods: false
```

Comment:
I'd name this `CheckForMethodsWithNoSideEffects` or something along those lines.

Reply:
I can imagine you can call some method that would not modify its receiver, but would still do something with side-effects. On the other hand whether this method will have a bang or not is unclear. That's why all of this is so tricky...